Sources/TuistKit/Services/GraphImplicitImportLintService.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+import Path
+import Tools
+import TuistCore
+import TuistLoader
+import TuistSupport
+import XcodeGraph
+
+enum GraphImplicitImportLintServiceError: LocalizedError {
+    case readFile
+
+    var errorDescription: String? {
+        switch self {
+        case .readFile:
+            "Error while reading file"
```

Comment:
Should we include the name of the file that we failed reading?

Reply:
Removed that error. Now Filehandler will notify about it